src/utils/stringUtils.ts

```diff
@@ -30,7 +37,7 @@ export class stringUtils {
      * }
     */
 
-    public static URLParse(string: string) {
+    public static urlParse(string: string) {
         let url = new URL(string);
         let searchParams_url = url.searchParams
```

Comment:
What the fuck? 

DONT MIX camelCase with snake_case!

Don't use Array<any> use any[];

Add a space between types!